aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/UploadPartTask.java

```diff
@@ -36,15 +36,6 @@
     private final TransferDBUtil dbUtil;
     private final NetworkInfoReceiver networkInfoReceiver;
 
-    public UploadPartTask(UploadPartRequest request,
```

Comment:
This will be a breaking change for developers who are using this constructor. Can you not remove this constructor?